### PR TITLE
dasel: Add version 1.8.0

### PR DESCRIPTION
**Pull request by littleli**
Use single selector DSL to query many of the data formats.

Demo: 
demo

### CODE REVIEW

File: bucket/dasel.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        [
+            "dasel_windows_amd64.exe",
+            "dasel"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/TomWright/dasel/releases/download/v$version/dasel_windows_amd64.exe"
```
Comment (Ash258) [inline bucket/dasel.json:22]:
```suggestion
                "url": "https://github.com/TomWright/dasel/releases/download/v$version/dasel_windows_amd64.exe#/dasel.exe"
```

File: bucket/dasel.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.0",
+    "description": "Query and update data structures from the command line. Comparable to jq/yq but supports JSON, TOML, YAML and XML with zero runtime dependencies",
+    "homepage": "https://github.com/TomWright/dasel",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TomWright/dasel/releases/download/v1.8.0/dasel_windows_amd64.exe",
```
Comment (Ash258) [inline bucket/dasel.json:8]:
```suggestion
            "url": "https://github.com/TomWright/dasel/releases/download/v1.8.0/dasel_windows_amd64.exe#/dasel.exe",
```

File: bucket/dasel.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            "hash": "5173274e90868d6d51a3ee580a2806650fd47107f7cee3b81c06ce92f4272a5f"
+        }
+    },
+    "bin": [
+        [
+            "dasel_windows_amd64.exe",
+            "dasel"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/dasel.json:17]:
```suggestion
    "bin": "dasel.exe",
```

File: bucket/dasel.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.8.0",
+    "description": "Query and update data structures from the command line. Comparable to jq/yq but supports JSON, TOML, YAML and XML with zero runtime dependencies",
```
Comment (Ash258) [inline bucket/dasel.json:3]: mention it is shortcut for `data selector`

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/dasel.json (hunk 4 above)
Comment (littleli) [inline bucket/dasel.json:3]:
```suggestion
    "description": "Dasel (short for data-selector) allows you to query and update data structures from the command line. Comparable to jq/yq but supports JSON, TOML, YAML and XML with zero runtime dependencies",
```

Comment (littleli) [reply bucket/dasel.json]: Done

File: bucket/dasel.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.8.0",
+    "description": "Dasel (short for data-selector) allows you to query and update data structures from the command line. Comparable to jq/yq but supports JSON, TOML, YAML and XML with zero runtime dependencies",
```
Comment (Ash258) [inline bucket/dasel.json:3]:
```suggestion
    "description": "DAta-SELector. Command line utility for querying and modifying data structures inside JSON, TOML, YAML, ...",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**